### PR TITLE
Add queues to crossbars.

### PR DESCRIPTION
**Pull request by grebe**
Also, fix up testing a bit.

### CODE REVIEW

File: src/main/scala/craft/Configs.scala [hunk 1]
```diff
@@ -66,6 +66,7 @@ object ConfigBuilder {
     }) ++ nastiTLParams(id)
   def nastiTLParams(id: String): Config = new Config(
     (pname, site, here) => pname match {
+      case XBarUsePortQueues => true
```
Comment (grebe) [inline src/main/scala/craft/Configs.scala:69]: @stevobailey I don't know if you think this is OK. It is kind of shady, but all the configs call it. It doesn't really belong here, but I think it requires the least amount of fiddling with other configs.

Comment (stevobailey) [reply src/main/scala/craft/Configs.scala:69]: I guess this is fine?

File: src/main/scala/craft/Configs.scala [hunk 2]
```diff
@@ -104,6 +105,7 @@ class WithCraft extends Config(
         AddrMapEntry(s"chan1", MemSize(0x200, MemAttr(AddrMapProt.RWX))))
     }
     case XBarQueueDepth => 2
+    case XBarUsePortQueues => true
```
Comment (stevobailey) [inline src/main/scala/craft/Configs.scala:108]: Confusingly, the XBarQueueDepth on the line above is unrelated, yes?
